### PR TITLE
June 2022 Release

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -5,10 +5,12 @@ REACT_APP_URL='https://portal.getpaidinbitcoin.com.au/'
...
 REACT_APP_DIGITAL_SOURCE='https://digitalid.com/sdk/app.js'
+REACT_APP_IDEM_URL='https://idem.com.au'
 # Development
 RREACT_APP_URL='https://testv2.getpaidinbitcoin.com.au/'
 REACT_APP_API_URL='https://testapi.getpaidinbitcoin.com.au'
 REACT_APP_BLOCKCYPHER_URL='https://api.blockcypher.com/v1/btc/test3'
 REACT_APP_USE_HD=true
 REACT_APP_DIGITAL_ID_CLIENT_ID='ctid0frFNICtT3eRqnvk6WaF3o'
 REACT_APP_DIGITAL_SOURCE='https://digitalid-sandbox.com/sdk/app.js'
+REACT_APP_IDEM_URL='https://idem.com.au'
```
Comment (xiangzhe0812) [inline .env.development:16]: we need to remove url for now, Idem not ready for production

File: .env.development [hunk 2]
```diff
@@ -5,10 +5,12 @@ REACT_APP_URL='https://portal.getpaidinbitcoin.com.au/'
 REACT_APP_USE_HD=false
 REACT_APP_DIGITAL_ID_CLIENT_ID=''
 REACT_APP_DIGITAL_SOURCE='https://digitalid.com/sdk/app.js'
+REACT_APP_IDEM_URL='https://idem.com.au'
```
Comment (xiangzhe0812) [inline .env.development:8]: we need to remove url for now, Idem not ready for production

**Review by xiangzhe0812 — COMMENTED** [review]
two comments, @bitcoinbrisbane  I can push changes if you agree

File: .env.development (hunk 1 above)
Comment (bitcoinbrisbane) [reply .env.development:16]: Its just the website, that says coming soon.  Im happy to leave it for now.

File: .env.development (hunk 2 above)
Comment (bitcoinbrisbane) [reply .env.development:8]: Its just the website, that says coming soon. Im happy to leave it for now.

**Pull request merged**